### PR TITLE
notes can now show in mytickets and unassigned views

### PR DESCRIPTION
**Pull request by isc-rsingh**

### CODE REVIEW

File: public/home.html [hunk 1]
```diff
@@ -102,9 +102,17 @@
     <!-- list of documents -->
     <div v-if="mode==='unassigned' || mode === 'mytickets' || mode === 'search'" class="container">
       <div class="row">
-        <div class="col-md-12">
+        <div class="col-md-10">
           <h2>{{ mode }} tickets <span class="badge" v-if="numDocs">{{ docs.length }}</span></h2>
         </div>
+        <div class="col-md-2">
+          <h2>
+            <button v-if="showNotes && mode === 'unassigned'" v-on:click="showNotes=false;unAssignedTickets" class="btn btn-primary align-bottom">Hide Notes</button>
```
Comment (vabarbosa) [inline public/home.html]: if you want you can replace these four buttons with a single button:
```
<button v-if="mode==='unassigned' || mode==='mytickets'" v-on:click="showNotes=!showNotes" class="btn btn-primary align-bottom">{{ showNotes ? 'Hide Notes' : 'Show Notes' }}</button>
```

File: public/home.html [hunk 2]
```diff
@@ -131,6 +139,30 @@
...
+                  <table class="table table-condensed">
+                    <tr v-if="doc.notes">
+                      <th colspan="6">what</th>
+                      <th>who</th>
+                      <th>when</th>
+                    </tr>
+                    <tr v-for="note in doc.notes">
+                      <td colspan="6">{{ note.note }}</td>
+                      <td>{{ note.who }}</td>
+                      <td>{{ note.time }}</td>
```
Comment (vabarbosa) [inline public/home.html]: maybe we can make the datetime a littl more user friendly:
```
<td>{{ new Date(note.time).toLocaleString() }}</td>
```
i would suggest doing the same in the Edit page as well

File: public/home.html (hunk 1 above)
Comment (isc-rsingh) [reply public/home.html]: Nice. I was wondering if the button text could be done with a ternary operator but didn't find that quickly.

**Comment by ptitzler** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/bb9178ab-039a-4b1b-bdfa-c5ab8e217c77 "sodashboard-wdp-infra"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/592bddce-9340-4b45-9afa-14c722dd7005 "sodashboard-wdp-infra") deployed [sodashboard](https://github.com/ibm-cds-labs/sodashboard/releases/tag/deploy-ibm-yp-us-south-Dev-Advocates-wdp-infrastructure-20170609-150946 "deploy-ibm-yp-us-south-Dev-Advocates-wdp-infrastructure-20170609-150946") to [wdp-infrastructure](https://console.ng.bluemix.net/apps/58ef4b5d-90eb-468a-bb05-3461093b239e?env_id=ibm:yp:us-south "ibm:yp:us-south:Dev-Advocates:wdp-infrastructure"), including commit f0be578eefca0dc4062ca3c023893ece9683e69e

**Comment by ptitzler** [comment]
Bluemix [toolchain](https://console.bluemix.net/devops/toolchains/afc93986-3917-48ae-8901-f070f6d109f7 "green-blue-toolchain-20170516074606603"): [Delivery Pipeline](https://console.bluemix.net/devops/pipelines/5ea1a679-3a40-49d5-9621-b48806ab22cc "green-blue-toolchain-20170516074606603") deployed [sodashboard](https://github.com/ibm-cds-labs/sodashboard/releases/tag/deploy-ibm-yp-us-south-ptitzler_org-dev-20170609-150955 "deploy-ibm-yp-us-south-ptitzler_org-dev-20170609-150955") to [dev](https://console.ng.bluemix.net/apps/42261ff4-0441-49e6-b978-900aa8255c10?env_id=ibm:yp:us-south "ibm:yp:us-south:ptitzler_org:dev"), including commit f0be578eefca0dc4062ca3c023893ece9683e69e

**Pull request merged**